bin/runtests.sh

```diff
@@ -139,15 +139,16 @@ for BUNDLE in $BUNDLES; do
         bash $BEFORE_SCRIPT
     fi
 
-    cd -
     comment "Running tests"
 
-    phpunit --configuration $BUNDLE
+    phpunit -c phpunit.xml.dist
 
     if [ $? -ne 0 ]; then
         echo $BUNDLE_NAME >> /tmp/failed.tests
     fi
 
+    cd -
+
```

Comment:
These changes were necessary in order to make the script work correctly under mac osx.